src/Graphics/src/Graphics/Colors.cs

```diff
@@ -95,6 +95,8 @@ public static class Colors
 		public static readonly Color DimGrey = DimGray;
 		/// <summary>Gets the system-defined color that has an ARGB value of <c>#FF1E90FF</c>.</summary>
 		public static readonly Color DodgerBlue = Color.FromUint(0xFF1E90FF);
+		/// <summary>Gets the system-defined color that has an ARGB value of <c>#0xFF512BD4</c>.</summary>
+		public static readonly Color DotNetPurple2024 = Color.FromUint(0xFF512BD4);
```

Comment:
Capitalization wise, that's consistent with priort art (e.g. `Microsoft.DotNet.*`). Adding the year is a way of future proofing, I guess.

Reply:
good thank you